src/smart-components/access-management/UserGroupsTable.tsx

```diff
@@ -135,7 +138,21 @@ const UserGroupsTable: React.FunctionComponent<UserGroupsTableProps> = ({
         group.workspaces || '?', // not currently in API
         formatDistanceToNow(new Date(group.modified), { addSuffix: true }),
         enableActions && {
-          cell: <ActionsColumn items={rowActions} />,
+          cell: (
+            <ActionsColumn
+              items={[
+                {
+                  title: intl.formatMessage(messages['usersAndUserGroupsEditUserGroup']),
+                  onClick: () => console.log('EDIT USER GROUP'),
+                },
+                {
+                  title: intl.formatMessage(messages['usersAndUserGroupsDeleteUserGroup']),
+                  onClick: () => handleDeleteModalToggle([group]),
+                },
+              ]}
+              rowData={group}
+            />
+          ),
```

Comment:
If the group has either `system=true` or `platform_default=true` flag enabled we can't show the kebab menu at all. These groups can't be changed or removed.

Reply:
![image](https://github.com/user-attachments/assets/fa470dde-fff2-4d24-95ba-347e651c1c39)
Actions are now disabled for `system` and `platform_default` rows, both individually and when one is included within a bulk selection.